nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/versioning/OrphanVersionCleanupListener.java

```diff
@@ -50,16 +49,7 @@
      */
     protected long getCommitSize() {
         ConfigurationService configurationService = Framework.getService(ConfigurationService.class);
-        String commitSize = configurationService.getProperty(DEFAULT_COMMIT_SIZE_PROP);
-        if (StringUtils.isBlank(commitSize)) {
-            return DEFAULT_COMMIT_SIZE;
-        }
-        try {
-            return Long.parseLong(commitSize);
-        } catch (NumberFormatException e) {
-            log.error("Invalid configuration property " + DEFAULT_COMMIT_SIZE_PROP, e);
-            return DEFAULT_COMMIT_SIZE;
-        }
+        return configurationService.getLong(DEFAULT_COMMIT_SIZE_PROP, DEFAULT_COMMIT_SIZE);
```

Comment:
Please keep blank behavior.

Reply:
It's now handle in `ConfigurationService`.